cpp/contracts/feesmanager.cpp

```diff
@@ -12,58 +12,56 @@ void feesmanager::init( name security_council ) {
+    auto allowance_itr = allowances.find(value.symbol.code().raw());
     
     if (allowance_itr == allowances.end()) {
-        allowances.emplace(get_self(), [&](auto& row) {
-            row.node = node;
-            row.token = token;
-            row.amount = amount;
+        allowances.emplace(get_self(), [&](auto& a) {
+            a.allowance_data = value;
         });
     } else {
-        allowances.modify(allowance_itr, get_self(), [&](auto& row) {
-            row.amount = amount;
+        allowances.modify(allowance_itr, get_self(), [&](auto& a) {
+            a.allowance_data = value;
         });
     }
 }
 
 // Increase allowance for a node
-void feesmanager::incallowance( name node, name token, uint64_t amount ) {
+void feesmanager::incallowance( name node, const asset& value ) {
     require_auth(owner);
 
-    allowances_table allowances(get_self(), get_self().value);
-    auto allowance_itr = allowances.find(node.value + token.value);
-    check(allowance_itr != allowances.end(), "No allowance set for this node");
+    allowances_table allowances(get_self(), node.value);
+    const auto& allowance_table = allowances.get(value.symbol.code().raw(), "No allowance set for this node");
 
-    allowances.modify(allowance_itr, get_self(), [&](auto& row) {
-        row.amount += amount;
+    allowances.modify(allowance_table, get_self(), [&](auto& a) {
+        a.allowance_data.amount += value.amount;
     });
 }
 
 // Withdraw function to withdraw tokens
-void feesmanager::withdrawto( name node, name token ) {
-    allowances_table allowances(get_self(), get_self().value);
-    auto allowance_itr = allowances.find(node.value + token.value);
-    check(allowance_itr != allowances.end(), "No allowance set for this node");
-    check(allowance_itr->amount >= 0, "Allowance is zero");
+void feesmanager::withdrawto( name node, const asset& token ) {
+    allowances_table allowances(get_self(), node.value);
+    const auto& allowance_table = allowances.get(token.symbol.code().raw(), "No allowance set for this node");
+    check(allowance_table.allowance_data.amount >= 0, "Allowance is zero");
 
-    allowances.modify(allowance_itr, node, [&](auto& row) {
-        row.amount = 0;
+    asset asset_data = asset(allowance_table.allowance_data.amount, token.symbol);
+
+    allowances.modify(allowance_table, node, [&](auto& a) {
+        a.allowance_data.amount = 0;
     });
 
     action(
         permission_level{get_self(), "active"_n},
-        token,
+        "eosio.token"_n,
         "transfer"_n,
-        std::make_tuple(get_self(), node, allowance_itr->amount, std::string("Withdraw"))
+        std::make_tuple(get_self(), node, asset_data, std::string("Withdraw"))
     ).send();
 }
 
-void feesmanager::withdrawto(name node, const std::vector<name>& tokens) {
+void feesmanager::withdrawto(name node, const std::vector<asset>& tokens) {
     for (const auto& token : tokens) {
         withdrawto(node, token);
```

Comment:
Double check this is working in a test because I'm not 100% sure it will, since `withdrawto` is declared as an action. 

If it doesn't you'll probably need to factor out the code within `withdrawto` to an "internal" function (which is not declared as ACTION) that can be called as needed.